tgui/packages/tgui/interfaces/DestinationTagger.js

```diff
@@ -0,0 +1,50 @@
+import { useBackend } from '../backend';
+import { Button, Section, Box, Flex } from '../components';
+import { LabeledListItem } from '../components/LabeledList';
+import { Window } from '../layouts';
+
+export const DestinationTagger = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    destinations,
+    selected_destination_id,
+  } = data;
+
+  let selected_destination = destinations[selected_destination_id - 1]
+
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <Section title="TagMaster 3.0">
+          <LabeledListItem>
+            <LabeledListItem label="Selected">{selected_destination.name ? selected_destination.name : "None"}</LabeledListItem>
```

Comment:
This is never null, is it? And if it is then a coder fucked up. Might want to write something that it is unexpected and/or to report it or something. Although leaving it like this is fine as well